src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -178,10 +178,6 @@ public void Listen()
             {
                 this.logger.LogDebug("Shutdown detected, stop accepting connections.");
             }
-            catch (Exception e)
-            {
-                this.logger.LogError("Exception occurred: {0}", e.ToString());
-            }
 
```

Comment:
wtf? revert this back to LogDebug as it was before
this change would crash node on shutdown

Reply:
Oh, right. 
Didn't see that we do .Wait() on it without handling

Reply:
>this change would crash node on shutdown  

Can you explain why?